package-lock.json

```diff
@@ -3522,95 +3522,6 @@
         "tslib": "^2.5.0"
       }
     },
-    "node_modules/@testing-library/dom": {
```

Comment:
hey are you sure this doesn't effect the code

Reply:
Yes